discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "inline",
+            "tutorials": "yes",
+            "examples": "yes",
+            "eventsAsync": "threaded",
```

Comment:
Oh, I see. Maybe it would be a good idea to make it possible to add a tool tip to this field (and `apiCallsAsync`), as the details will surely vary between libs.